api/api.go

```diff
@@ -37,6 +39,43 @@ type SubmissionInfo struct {
 	State string `json:"state"`
 }
 
+// StatusInfo contains information about a user's status on a particular language track.
+type StatusInfo struct {
+	Language string `json:"track_id"`
+	Recent   struct {
+		Problem   string `json:"problem"`
+		Submitted string `json:"submitted_at"`
```

Comment:
I would call this `At` or `SubmittedAt` -- my naming convention for timestamps is `*_at` and my naming convention for dates is `*_on`.